net/portmapper/pcp.go

```diff
@@ -34,41 +38,89 @@ const (
 	pcpTCPMapping = 6  // portmap TCP
 )
 
-// pcpMapRequest generates a PCP packet with a MAP opcode.
```

Comment:
restore comment. new version lacks a comment.

you can then document that lifetimeSec 0 means delete.